WordPress/gutenberg
# Mobile: BottomSheet design tweaks

## Description

This PR adds some design options to the BottomSheet and its Cells.

- A new option for a cell with just one Label with centered text.
- A possibility to add custom styles to both Label and Value text.
- A new option for adding an icon to the cell.
- A possibility to completely hide the BottomSheet header.
- A new separator with an option for not showing it (helpful for the last cell).

<img width="391" alt="screen shot 2019-02-01 at 10 59 02 am" src="https://user-images.githubusercontent.com/9772967/52116022-6659bc80-2610-11e9-8f75-57ff0c6cb3f6.png">

**To test:**

- Checkout the related `gutenberg-mobile` PR: (https://github.com/wordpress-mobile/gutenberg-mobile/pull/533)
- Run the example project.
- Tap on a image block with image.
- Tap on the image settings button (the wheel icon in the inline toolbar).
- Check that the BottomSheet displays as shown in the screenshot (with just one Alt text cell).
- If you feel like it, play with different configurations for the BottomSheet and the Cells.
